### DISCUSSION TITLE
ReplaceTrack

### DISCUSSION THREAD

**Discussion by amir2006544** [aiortc/aiortc#1022] [Q&A]
**Hi everyone!**
How can I use the track change method inside the library?

It is not explained in detail anywhere and there is no such method in the library

**Comment by jlaine** [comment]
Sorry but replacing tracks is not currently supported.

**Reply by iandresolares** [reply]
Hi, I assume this is not yet supported right? Are there any plans to support it at any time? What is the main barrier?

thanks!
